apps/meteor/client/views/admin/manageSubscription/ManageSubscriptionPage.tsx

```diff
@@ -0,0 +1,172 @@
+import { Box, Button, ButtonGroup, Callout, Grid, Skeleton, Throbber } from '@rocket.chat/fuselage';
+import { useRouter } from '@rocket.chat/ui-contexts';
+import { t } from 'i18next';
+import React, { memo, useEffect } from 'react';
+
+import Page from '../../../components/Page';
+import { useIsEnterprise } from '../../../hooks/useIsEnterprise';
+import { useLicense } from '../../../hooks/useLicense';
+import { useRegistrationStatus } from '../../../hooks/useRegistrationStatus';
+import { PlanName, getPlanName } from '../../../lib/utils/getPlanName';
+import GetMoreWithEnterprise from './components/GetMoreWithEnterprise';
+import UpgradeButton from './components/UpgradeButton';
+import ActiveSessionsCard from './components/cards/ActiveSessionsCard';
+import AppsUsageCard from './components/cards/AppsUsageCard';
+import CountMACCard from './components/cards/CountMACCard';
+import CountSeatsCard from './components/cards/CountSeatsCard';
+import FeaturesCard from './components/cards/FeaturesCard';
+import MACCard from './components/cards/MACCard';
+import PlanCard from './components/cards/PlanCard';
+import SeatsCard from './components/cards/SeatsCard';
+import { useWorkspaceSync } from './hooks/useWorkspaceSync';
+import { CONTACT_SALES_LINK } from './utils/links';
+
+const ManageSubscriptionPage = () => {
+	const router = useRouter();
+	const { data } = useIsEnterprise();
+	const { isRegistered, isLoading: isRegisteredLoading } = useRegistrationStatus();
+	const { data: licensesData, isLoading: isLicenseLoading, refetch: refetchLicense } = useLicense();
+	const syncLicenseUpdate = useWorkspaceSync();
+
+	const { subscriptionSuccess } = router.getSearchParameters();
+
+	const { activeModules = [], license, limits } = licensesData?.data || {};
+	const isEnterprise = data?.isEnterprise || false;
+	const isTrial = license?.information?.trial ?? false;
+
+	const marketplaceAppsLimit = limits?.marketplaceApps?.max;
+	const privateAppsLimit = limits?.privateApps?.max;
+	const monthlyActiveContactsLimit = limits?.monthlyActiveContacts?.max;
+
+	const plan = getPlanName(isEnterprise, activeModules, isTrial);
+
+	const getUpgradeButton = () => {
+		if (plan === PlanName.COMMUNITY || plan === PlanName.STARTER) {
+			return <UpgradeButton primary i18nKey='Upgrade_to_Pro' mis={8} />;
+		}
+
+		if (plan === PlanName.PRO || plan === PlanName.PRO_TRIAL) {
+			return (
+				<Button is='a' type='button' primary external href={CONTACT_SALES_LINK} mis={8}>
+					{t('Start_Enterprise_trial')}
+				</Button>
+			);
+		}
+
+		return null;
+	};
+
+	const getHeaderButtons = () => {
+		return (
+			<ButtonGroup>
+				{isRegistered && (
+					<Button
+						icon={syncLicenseUpdate.isInitialLoading || syncLicenseUpdate.isRefetching ? undefined : 'reload'}
+						onClick={() => handleSyncLicenseUpdateClick()}
+					>
+						{syncLicenseUpdate.isInitialLoading || syncLicenseUpdate.isRefetching ? (
+							<Throbber size='x12' inheritColor />
```

Comment:
Can't we just disable the `Button` and change the label?

Reply:
Design stuff... They will work on a loading button to add to fuselage